youtube_dl/extractor/cba.py

```diff
@@ -0,0 +1,104 @@
+        }
+    }
+    _FORMATS = {
+        'audio/ogg': {'id': '1', 'ext': 'ogg', 'preference': 100},
+        'audio/mpeg': {'id': '2', 'ext': 'mp3', 'preference': 50}
+    }
+    _API_KEY = None
+
+    def __init__(self, *args, **kwargs):
+        try:
+            self._API_KEY = os.environ["CBA_API_KEY"]
```

Comment:
AFAIR there are no extractors that use personal API keys since there is almost always a way to workaround mimicking browser's behavior without it.
It should not use env at all. Any integration should use generic interface but not a zoo of system wide env variables.